lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+  border-bottom: 1px solid theme-color("primary");
+  box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.1);
+  background: $white;
+
+  /*
+    Logo and course identification block
+  */
+  .header-logo {
+    display: inline;
+
+    a {
+      @include float(left);
+      @include margin($baseline/2, 0, 0, $baseline);
+      display: block;
+
+      .logo {
+        @include float(left);
+        height: 40px;
+      }
+    }
+
+    .course-header {
+      @include float(left);
+      @include margin($baseline*1.25, 0, 0, $baseline);
+      color: theme-color("dark");
+      font-size: 16px;
+      line-height: 1em;
+
+      .provider {
+          font-weight: bold;
+      }
+
+      // Hide the course header for smaller screen sizes
+      @include media-breakpoint-down(md) {
+        display: none;
+      }
+    }
+  }
+
+/*
+  Main navigation buttons for Courses, programs, profile and explore buttons.
+
+  There are two stylings for the two circumstances, first the mobile, followed
+  by the desktop styling
+*/
+  // Desktop styling
+  @include media-breakpoint-up(md) {
+    .nav-links {
+      .nav-item {
+        margin: 0 $baseline;
+
+        a {
+          text-decoration: none;
+        }
+      }
+
+      .main {
+        @include float(left);
+        @include margin($baseline, 0, 0, $baseline);
+        text-transform: uppercase;
+
+        .nav-item {
+          @include float(left);
+        }
+
+        .nav-tab {
+          text-transform: none;
+          padding: 0;
+          cursor: pointer;
+          margin: 0;
+
+          a {
+            color: theme-color("secondary");
+            padding: $baseline/2 $baseline*1.25 $baseline*0.75;
+            font-weight: 600;
+            display: inline-block;
+            margin-bottom: -1*$baseline/2;
+            border-bottom: 2px solid transparent;
+            cursor: pointer;
+
+            &.active, &:hover {
+              border-bottom-color: theme-color("dark");
+            }
+
+            &:hover {
+              cursor: pointer;
+              border-bottom-color: theme-color("primary");
+            }
+          }
+        }
+      }
+
+      .secondary {
+        @include float(right);
+        margin: $baseline*0.6 $baseline 0 0;
+
+        // All navigation items
+        .nav-item {
+          display: inline-block;
+          padding: $baseline/2 0 0;
+          margin: 0 $baseline/2;
+          font-size: 14px;
+
+          a {
+            color: theme-color("dark");
+            font-weight: bold;
+          }
+        }
+
+
+        // Sign in, Register and Shopping Cart buttons
+        .btn {
+          margin: 0 $baseline/4 0;
+          border: 1px solid theme-color("primary");
+          padding: $baseline/4 $baseline;
+          border-radius: 5px;
+          cursor: pointer;
+          text-decoration: none;
+        }
+
+        .sign-in-btn, .shopping-cart {
+          background: theme-color("primary");
+          color: $white;
```

Comment:
Use `theme-color("inverse")` for all the places that you currently have `$white`.